content/lang/collections/fieldterms/occurrenceterms.es.php

```diff
@@ -2,93 +2,93 @@
 /*
 ------------------
 Language: Español (Spanish)
```

Comment:
Weird. So these translations were just missing from Spanish but not from French or Portuguese? I wonder why... @ksorellana, was this intentional for some reason I'm not thinking about?